client/settings/express-checkout/test/index.test.js

```diff
@@ -54,7 +54,7 @@ describe( 'ExpressCheckout', () => {
 
 		usePlatformCheckoutEnabledSettings.mockReturnValue(
 			getMockPlatformCheckoutEnabledSettings(
-				false,
+				true,
```

Comment:
I tweaked this just for the purpose of the tests: before, both WooPay and Link were disabled. So clicking on both of them would have displayed the confirmation modal.
With these changes, this test shows only one of them enabled at the same time (and the expectations on the handlers are still working ok).